syncer/metrics/metrics.go

```diff
@@ -209,13 +237,38 @@ var (
 			Name:      "heartbeat_update_error",
 			Help:      "number of error when update heartbeat timestamp",
 		}, []string{"server_id"})
+
+	FinishedTransactionTotal = metricsproxy.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "dm",
+			Subsystem: "syncer",
+			Name:      "finished_transaction_total",
+			Help:      "total number of finished transaction",
+		}, []string{"task", "worker", "source_id"})
+
+	ReplicationTransactionBatch = metricsproxy.NewGaugeVec(
```

Comment:
as discussed this also should be histogram

Reply:
fixed in https://github.com/pingcap/dm/pull/1778/commits/b143bb367b9d493d28b632d47fd1b8c4a101b0df